src/core/annotation.js

```diff
@@ -772,10 +845,12 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
     if (!isDict(appearanceStates)) {
       return;
     }
+
```

Comment:
Revert the newline changes here and a few lines below. They are unrelated to your patch.

Reply:
Done, thank you